### ISSUE TITLE
Dokumentation

### ISSUE THREAD

**Issue by thkl** [thkl/hap-homematic#117]
möchte vielleicht jemand hier etwas Dokumentation entwerfen ?

**Comment by mike4president** [comment]
Wenn ich helfen kann, gerne...

**Comment by thkl** [comment]
ich hätte die Wiki Funktion hier dafür vorgesehen. Es geht im Prinzip um kurze Einträge, wie verschiedene Sachen gelöst werden können. Zb was beim Garagentor zu beachten ist, oder wenn Services mehrere Parameter haben, was diese bewirken.

**Comment by mike4president** [comment]
Ich könnte das mit der Alarmanlage beschreiben.

**Comment by ChrWi** [comment]
Ich würde auch helfen - versuche gerade das Thema Garagentor zu verstehen, was ich bei homebridge-homematic ohne Probleme laufen haben. Wie bekomme ich den Typ "Schalter" als default geändert (den ich auch für HmIP-PSM bekomme, während homebridge-homematic die Steckdose zeigt)

**Comment by thkl** [comment]
Die Settings fürs Tor sind die gleichen. Hier gibt’s nen länglichen Issue zum Tor. Da wurde das alles zusammengesucht und ausprobiert.

**Comment by ChrWi** [comment]
Ich orientiere mich. HmIP-PSM habe ich gelöst. Insgesamt sehr intuitive Anwendung. 
Habe ich die Berechtigung für Einträge im Wiki?

**Comment by thkl** [comment]
Invite folgt 🤓

**Comment by ChrWi** [comment]
Prima. Habe das Garagentor nun unter besondere Geräte entdeckt. Kann das also mit Vorlage der funktionierenden Konfig im Homebridge angehen und parallel dokumentieren.

**Comment by ChrWi** [comment]
Vielleicht blöde Frage: wie kann ich im Wiki Bilder hochladen/bereitstellen?

**Comment by dkrizic** [comment]
Einfach im Browser reinpasten.

**Comment by ChrWi** [comment]
Nee, leider nicht. Das geht hier in den Issues, aber das Wiki sieht bei mir "schlicht" aus.

**Comment by thkl** [comment]
Ja, im Wiki sind leider keine Bilder ein"paste"bar wie in Issues. Daher lagere ich die alle in doc/wiki ... wenn Du alle Bilder zusammen hast mail sie mir. Ich stell sie ins Repo und dann kann man sie von dort verlinken.

**Comment by thkl** [comment]
Ich habe mal einen Beitrag dazu gebaut, was eine Instanz ist und worum es da überhaupt geht und  den in den Pages verlinkt

**Comment by sailor79** [comment]
Hallo Thomas,
ich würde bei der Doku unterstützen wollen und mal was Struktur reinbringen. Gruß Marc

**Comment by thkl** [comment]
> Hallo Thomas,
> ich würde bei der Doku unterstützen wollen und mal was Struktur reinbringen. Gruß Marc

😍

**Comment by sailor79** [comment]
Hallo Thomas, gibt es hier noch eine zusätzliche 1to1 Kommunikationsmöglichkeit?

**Comment by sailor79** [comment]
So, der Anfang ist gemacht, siehe "Home". (Struktur und bisschen content), bitte mal checken ob das so grundsätzlich passt. Ich würde erstmal weiter mit den Anchors arbeiten. Wenn der Content dann doch überläuft wenn erstmal das meiste da ist , kann man Teile wieder in Subpages ausgliedern.

**Comment by thkl** [comment]
Oh coole Sache, vielen Dank. Ich habe nur eine Kleinigkeit zum CCU Addon ergänzt. Github erzeugt bei einem Release immer ein Paket des Codes. Das wird aber auf der CCU nicht benötigt, sondern nur der eigentliche Installer.

**Comment by sailor79** [comment]
Hallo Thomas, Du hast bestimmt eine Liste aller (kontextsensitiven) Services, z.B. "HomeMaticWindowsAccessory - This service provides a window device in HomeKit based on a CCU contact." ? Dann muss ich die nicht alles einzeln erforschen, sofern ich überhaupt alle möglichen Geräte habe.

**Comment by thkl** [comment]
Das ist recht einfach aus dem Code zu ermitteln. Jedes File hier https://github.com/thkl/hap-homematic/tree/master/lib/services stellt einen Service dar. Also fast jedes. Es muss so aussehen Homematic*Accessory.js
Und im File steht die Description in der statischen Methode serviceDescription()

**Comment by thkl** [comment]
> Hallo Thomas, gibt es hier noch eine zusätzliche 1to1 Kommunikationsmöglichkeit?

https://join.slack.com/t/neuerworkspac-km37354/shared_invite/zt-hnlaon12-VoVmXUX7lzgOWYCnzb6QnA

**Comment by sailor79** [comment]
Done for this weekend. Please check.

**Comment by thkl** [comment]
Great, i very appreciate the work u've done.

**Comment by ChrWi** [comment]
@sailor79 wenn ich meine damals in Deutsch geschriebenen HowTo übersetzen/umschreiben soll, können wir uns gern abstimmen

**Comment by sailor79** [comment]
> @sailor79 wenn ich meine damals in Deutsch geschriebenen HowTo übersetzen/umschreiben soll, können wir uns gern abstimmen

Gerne. Einfach machen. Ich schlage vor du schreibst die englische Version unter die deutsche, ich kopiere mir sie dann rüber in entsprechender Formatierung. Bezieht sich auf eine Variante Garagentor, richtig?

**Comment by ChrWi** [comment]
Korrekt - eine Variante Garagentor. Die Seite mit der schaltbaren Steckdose kann ich dann auch gerade übersetzen. 
ABER: wo bekomme ich die Sprache der UI auf EN umgestellt: HM UI, HAP oder Browsersprache?

**Comment by thkl** [comment]
Die Sprache hängt am Browser.

**Comment by ChrWi** [comment]
Bei mir am System (macOS) - habe ich in 15 Jahren Mac bisher nicht gebraucht.

**Comment by ChrWi** [comment]
@sailor79 die beiden wiki-pages für das Garagentor und die Steckdose habe ich kurzerhand übersetzt. 
Wir können die beiden Seiten nach Übernahme in deine (sehr gelungene!) Struktur gerne deaktivieren.
@thkl die neuen Screenshots habe ich per Mail geschickt

**Comment by thkl** [comment]
@ChrWi files liegen in https://github.com/thkl/hap-homematic/tree/master/doc/wiki/garage_en

**Comment by hillinator** [comment]
Hallo @thkl , HomeMaticSPVideoDoorBellAccessory scheint ja wohl zu funktionieren. Ich habe ebenfalls den HM-Sen-DB-PCB in der config hinterlegt, aber in Home kann ich das Ding nicht konfigurieren. Hast du mal eine Beispiel Konfig? Evtl. kann das ja mit die WiKi ;)

**Comment by sailor79** [comment]
@ChrWi: Sorry, dass ich so spät Antworte. Habe Deinen Input so vor einiger Zeit übernommen. Besten Dank für Deine Hilfe :-)

**Comment by ChrWi** [comment]
Hallo @thkl 
ich habe die "Internals" angepasst, da sich die Settings mittlerweile erweitert haben. Bitte schaue insb. bei Cache und Watchdog einmal drauf. Und tausche bitte das bestehende Bild gegen den angehängten Screenshot aus. Danke Dir!
<img width="791" alt="Screenshot 2021-09-06 131337" src="https://user-images.githubusercontent.com/52505025/132210641-c85b0f8f-5142-4466-b63c-e3f321eba591.png">
